tests/unit/test_fvm_multi.cpp

```diff
@@ -265,21 +266,21 @@ TEST(fvm_multi, mechanism_indexes)
 
     cell c;
     auto soma = c.add_soma(12.6157/2.0);
-    soma->add_mechanism(hh_parameters());
+    soma->add_mechanism("hh");
 
     // add dendrite of length 200 um and diameter 1 um with passive channel
     c.add_cable(0, section_kind::dendrite, 0.5, 0.5, 100);
     c.add_cable(1, section_kind::dendrite, 0.5, 0.5, 100);
     c.add_cable(1, section_kind::dendrite, 0.5, 0.5, 100);
 
     auto& segs = c.segments();
-    segs[1]->add_mechanism(pas_parameters());
-    segs[2]->add_mechanism(hh_parameters());
-    segs[3]->add_mechanism(pas_parameters());
+    segs[1]->add_mechanism("pas");
+    segs[2]->add_mechanism("hh");
+    segs[3]->add_mechanism("pas");
 
     for (auto& seg: segs) {
         if (seg->is_dendrite()) {
-            seg->mechanism("membrane").set("r_L", 100);
+            seg->rL = 100;
```

Comment:
The membrane-specific `rL` and `cm` are now set differently to other parameters.
They are not associated with a specific mechanism, but this approach is less "consistent" with the method for setting mechanism parameters.

Reply:
It's true, but I believe that's as it should be: they mean different things and play a different role in the model description, and ultimately, are used in different parts of the solver.